pkg/kvstore/etcd.go

```diff
@@ -788,7 +791,7 @@ func connectEtcdClient(ctx context.Context, config *client.Config, cfgPath strin
 
 	go ec.statusChecker()
 
-	ec.controllers.UpdateController("kvstore-etcd-session-renew",
+	ec.controllers.UpdateController(makeSessionName(etcdSessionRenewNamePrefix, opts.ClusterName),
```

Comment:
`opts` can be `nil` here, that's probably why the integration tests are failing: https://github.com/cilium/cilium/actions/runs/4240559572/jobs/7371405312

Reply:
ofcourse, thanks